### ISSUE TITLE
Doctrine Bundle: Hint in generated .env file does not work

### ISSUE THREAD

**Issue by derrabus** [symfony/recipes#151]
Given, I've bootstrapped a new application this way:

```bash
composer create-project symfony/skeleton demo
cd demo
composer require cli orm
```

If I now look into my .env file, I see the following section:

```
###> doctrine/doctrine-bundle ###
# Format described at http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url
# For a sqlite database, use: "sqlite:///%kernel.project_dir%/var/data.db"
# Set "serverVersion" to your server version to avoid edge-case exceptions and extra database calls
DATABASE_URL="mysql://root@127.0.0.1:3306/symfony?charset=utf8mb4&serverVersion=5.7"
###< doctrine/doctrine-bundle ###
```

Oh, that sounds like a good idea. So let's try that SQLite suggestion and set:

```
DATABASE_URL="sqlite:///%kernel.project_dir%/var/data.db"
```

![bildschirmfoto 2017-08-11 um 10 35 25](https://user-images.githubusercontent.com/1506493/29206156-eec9e81c-7e80-11e7-8a53-88440afcc4f3.png)

I'm not sure if kernel parameters are supposed to work inside environment variables (I guess not). Of course, it works fine if I replace `%kernel.project_dir%` with the real path to the project, but I don't see how I should know that I'm supposed to do that.

A user that is new to symfony would be lead directly into an error message that is not helpful.

**Comment by javiereguiluz** [comment]
You are right. See https://github.com/symfony/symfony/issues/23527 and https://github.com/symfony/symfony-docs/pull/8181 too.

**Comment by weaverryan** [comment]
@javiereguiluz I think we can/should update the docs now with an example of `resolve`: https://github.com/symfony/recipes/pull/204/files

And I believe #204 will fix this.

**Comment by derrabus** [comment]
@weaverryan #204 fixes this for Symfony 3.4, not for Symfony 3.3.

**Comment by fabpot** [comment]
As there is no way to fix it in 3.3, and because #204 is the solution, I'm closing this issue.
